GetStream/stream-chat-android
# [CAS-1496] Fix attachment previews in pinned message list

https://stream-io.atlassian.net/browse/CAS-1496

### 🎯 Goal

Fix how messages with only image attachments are rendered on the pinned message list screen.

### 🎨 UI Changes

| Before | After |
| --- | --- |
| ![photo_2021-12-20 16 06 55](https://user-images.githubusercontent.com/9600921/146771781-280e8e83-cdba-4951-b42c-fdc26588fec3.jpeg) | ![photo_2021-12-20 16 07 44](https://user-images.githubusercontent.com/9600921/146771869-994b9a85-104f-47eb-bdab-d0eae2672f13.jpeg) |
| ![photo_2021-12-20 16 06 52](https://user-images.githubusercontent.com/9600921/146771786-18189fa7-594b-4d5f-9123-730470e56cf6.jpeg) | ![photo_2021-12-20 16 07 47](https://user-images.githubusercontent.com/9600921/146771862-11bde4ab-43ff-4ce5-9ed1-74011aeae76a.jpeg) |

### 🧪 Testing

1. Pin a message with image only image attachments
2. Navigate to the pinned message list screen
3. Observe that the previously pinned message has the correct message preview

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_
